packages/browser-crawler/src/internals/browser-crawler.ts

```diff
@@ -509,6 +509,15 @@ export abstract class BrowserCrawler<
             }
         }
 
+        if (!this.requestMatchesEnqueueStrategy(request)) {
+            this.log.debug(`Skipping request ${request.id} (${request.url}) because it does not match the enqueue strategy.`);
```

Comment:
here its gonna be always because of redirect, right? we should mention it here and print the `loadedUrl` too i guess

also how come the fix is only in browser crawler, the same issue is present with http too, isnt it?

Reply:
> also how come the fix is only in browser crawler, the same issue is present with http too, isnt it?

Check the diff for the http-crawler file

Reply:
right, overlooked that one and was expecting a change in basic crawler but that does not fire requests...